js/main.js

```diff
@@ -122,6 +159,10 @@ async function main() {
   if (CURRENT_PAGE == "/gacha.html") {
     GACHA_BUTTON.onclick = (event) => pullAndRenderCards(CARDS_PER_PULL);
   }
+  if (CURRENT_PAGE == "/gacha.html" || CURRENT_PAGE == "/collection.html") {
+    DETAILS_DIALOG_CLOSE.onclick = (event) =>
+      DETAILS_DIALOG.classList.add("hidden");
+  }
```

Comment:
This part can be made clearer using a switch statement with fall-through instead of ifs.